lib/text/text_utils.js

```diff
@@ -180,6 +181,49 @@ shaka.text.Utils = class {
     return uniqueCues;
   }
 
+  /**
+   * @param {!VTTCue} vttCue
+   * @return {?shaka.text.Cue}
+   */
+  static mapNativeCue(vttCue) {
```

Comment:
Since we have `mapShakaCueToNativeCue`, I think a better name would be: `mapNativeCueToShakaCue`. I also think that we could extract this to another PR that is of the chore type and we can migrate it to older branches.